ui/src/common/ArtistLinkField.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-admin'
-import { useAlbumsPerPage } from './index'
 import { withWidth } from '@material-ui/core'
 
 export const useGetHandleArtistClick = (width) => {
```

Comment:
Width is not required anymore, so it does not make sense to have this hook. Please remove it

Reply:
Thank you for the review! I'll remove it.

Reply:
The width might be required; in case of failed response, the page would be redirected to the page with Artist filters,
```
  const [perPage] = useAlbumsPerPage(width)

  <Redirect
          to={`/album?filter={"artist_id":"${record?.id}"}&order=ASC&sort=maxYear&
              displayedFilters={"compilation":true}&perPage=${perPage}`}
   />
```
Or the page should quit after logging the error
Thoughts?

Reply:
See my comment below about `getArtist`. You won't be using it, so there's no need for that `useEffect` and even less for the try...catch block with the redirect.